templates/prometheus.yaml

```diff
@@ -90,6 +108,23 @@ spec:
           thanos:
             baseImage: thanosio/thanos
             version: v0.4.0
+          externalLabels:
+            cluster: $(CLUSTER_ID)
+          containers:
+            - name: get-cluster-id
+              image: mesosphere/kubeaddons-addon-initializer:v0.0.12
+              command: ["/bin/bash", "-c", "addon-initializer prometheus && while true; do sleep 86400; done"]
```

Comment:
```suggestion
              command: ["/bin/bash", "-c", "addon-initializer prometheus && sleep infinity"]
```

Reply:
seeing `sleep: invalid number 'infinity'` - doesn't look like `infinity` is supported, reverting

Reply:
That took way longer than I'd hoped, but v0.0.13 is ready and fixes sleep.

Reply:
cool, thanks for doing that I'll make the change

Reply:
@joejulian I've made this change - mind re-reviewing?